sql-plugin/src/main/scala/com/nvidia/spark/rapids/DateUtils.scala

```diff
@@ -42,6 +46,47 @@ object DateUtils {
 
   val ONE_DAY_MICROSECONDS = 86400000000L
 
+  val EPOCH = "epoch"
+  val NOW = "now"
+  val TODAY = "today"
+  val YESTERDAY = "yesterday"
+  val TOMORROW = "tomorrow"
+
+  def specialDatesDays: Map[String, Int] = {
+    val today = DateTimeUtils.currentDate(ZoneId.of("UTC"))
```

Comment:
I realize this isn't specific to this change, but should we be using a hardcoded UTC timezone here?  I realize it works because we only support UTC for timestamps, but what if the user is using dates (not timestamps) in a different timezone than UTC?

Reply:
That's a good question. I've been reading a blog post about time zone handling in Spark 3.0.0 [1] and it does state that `now` should use the session time zone, so it does seem better to remove the explicit UTC timezone. I will try this out.

[1] https://databricks.com/blog/2020/07/22/a-comprehensive-look-at-dates-and-timestamps-in-apache-spark-3-0.html